homebrew.js

```diff
@@ -0,0 +1,102 @@
+let casks = []
+
+fetch('https://raw.githubusercontent.com/AdoptOpenJDK/homebrew-openjdk/master/README.md', {})
+  .then(
+    function(response) {
+      if (response.status !== 200) {
+        console.log('Looks like there was a problem. Status Code: ' +
+          response.status);
+        return;
+      }
+      response.text().then(function(data) {
+        // Grab all the cask files from the README.md
+        const regex = /\b(\w*adoptopenjdk\w*[-]?\w*)\b/gm
+        casks = data.match(regex);
+
+        fetch('https://formulae.brew.sh/api/analytics/cask-install/365d.json', {})
```

Comment:
Can some of this be split into actual functions?

Reply:
I kind of wanted to keep this as a single logical function